test/flycheck-test.el

```diff
@@ -3910,10 +3910,19 @@ Why not:
            (geiser-mode))))
     (add-hook 'scheme-mode-hook setup-geiser)
     (unwind-protect
-        (flycheck-ert-should-syntax-check
-         "language/chicken.scm" 'geiser-mode
-         '(2 nil warning "in procedure call to `g1', expected a value of type `(procedure (* *) *)' but was given a value of type `number'"
-             :checker scheme-chicken))
+        (progn
+          (flycheck-ert-should-syntax-check
+           "language/chicken/error.scm" 'geiser-mode
+           '(2 nil warning "in procedure call to `g1', expected a value of type `(procedure (* *) *)' but was given a value of type `number'"
+               :checker scheme-chicken)
+           (flycheck-ert-should-syntax-check
+            "language/chicken/syntax-error.scm" 'geiser-mode
+            '(1 nil warning "invalid sharp-sign read syntax: #\n"
+                :checker scheme-chicken))
+           (flycheck-ert-should-syntax-check
+            "language/chicken/syntax-error-2.scm" 'geiser-mode
+            '(1 nil warning "not enough arguments\n\n	(define)\n\n	Expansion history:\n\n	<syntax>          (##core#begin (define))\n	<syntax>          (define)	<--"
+                :checker scheme-chicken))))
```

Comment:
Please do not use more than one of these assertions per test, so please split this test case into multiple test cases